docs/OptimizationTips.rst

```diff
@@ -40,7 +40,10 @@ four different optimization levels:
 
 In the Xcode UI, one can modify the current optimization level as follows:
 
-...
+For the whole project,
+Select project in `project and targets list`, then add `-Ounchecked` in `Other Swift Flags` under `Swift Compiler - Custom Flags`.
```

Comment:
Nit: It seems like this wants a section on attaching *any* of the flags mentioned above to a compile command line.  Perhaps

```
In the Project Navigator, select the Project to enter the Project Editor. From there, an optimization setting can be applied to every target in the project by changing the "Optimization Level" field under the `Build Settings` header.

To apply a custom optimization level to a singular target, select that target in the Project Editor and override the "Optimization Level" field under its `Build Settings` header.

If a given optimization level is not available in the UI, its corresponding flag can be specified by selecting the `Other...` level in the "Optimization Level" dropdown.
```